### PR TITLE
Added workflow to remove stale, incomplete issues

### PR DESCRIPTION
**Pull request by dr7ana**
New github workflow to remove incomplete issues that are abandoned by the submitting user.

To be deleted, an issue must be:

1. Tagged as "incomplete", request made for more information
2. No response after being tagged as incomplete for 14 days, issue is marked "stale"
3. After 7 more days of no response, issue is deleted

### CODE REVIEW

**Comment by tewinget** [comment]
You accidentally included a version bump of the ngtcp2 submodule in this.

File: .github/workflows/clean_issues.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
```
Comment (jagerman) [inline .github/workflows/clean_issues.yml]: I think we can drop this permission since we aren't applying the PR-closing behaviour here.

Comment (dr7ana) [reply .github/workflows/clean_issues.yml]: fixed! @tewinget also fixed the ngtcp2 bump

**Review by jagerman — APPROVED** [review]

**Comment by jagerman** [comment]
LGTM; let's merge it and see how it goes.

**Pull request merged**